hack/functest.sh

```diff
@@ -8,6 +8,7 @@ if [ -n "$OPENSHIFT_BUILD_NAMESPACE" ]; then
 	echo "Openshift CI detected, deploying using image $CATALOG_FULL_IMAGE_NAME"
 fi
 
+export GOFLAGS=""
```

Comment:
I would put this export within this block just before the first "go get":

````
if ! [ -x "$GINKGO" ]; then
	echo "Retrieving ginkgo and gomega build dependencies"
	go get github.com/onsi/ginkgo/ginkgo
	go get github.com/onsi/gomega/...
else
````

Reply:
Done